### PR TITLE
Project - DroneWorld

### PR DESCRIPTION
**Pull request by yashaswini-slu**
added the description details for drone world

### CODE REVIEW

File: docs/droneworld/about.md [hunk 1]
```diff
@@ -0,0 +1,54 @@
+---
+title: Drone World
+custom_edit_url: https://github.com/oss-slu/DroneWorld
+---
+
+## Overview
+DroneWorld is a platform for testing small unmanned aerial systems (sUAS) applications by simulating realistic test scenarios based on specified requirements.DroneWorld allows users to configure test scenarios by specifying environmental conditions (e.g. weather, terrain), sUAS capabilities (sensors, hardware), and test properties derived from system requirements. It generates a realistic 3D simulation environment using digital shadow models and runs the user's sUAS application within it. DroneWorld monitors and collects data to check if safety properties are violated. It produces test reports analyzing simulation results.
```
Comment (kate-holdener) [inline docs/droneworld/about.md:7]: Missing a space after the first sentence. Also, would be good to emphasize the value of this software to the end-users. Just a sentence or two describing why this software is important/valuable/useful, something to get the readers excited about it.

**Review by kate-holdener — CHANGES_REQUESTED** [review]

File: docs/droneworld/about.md [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+### Information
+
+- **Source Code:**  https://github.com/oss-slu/DroneWorld
+- **Client** Dr.Ankit Agrawal - https://github.com/ankyAgrawal
+- **Current Tech Lead:** Yashaswini Shivalingaiah (https://github.com/yashaswini-slu, https://www.linkedin.com/in/yashaswini-shivalingaiah-467a9652/)
+- **Developers:**
+    - Kaleb Yu (https://github.com/kalyus)
+    - Szymon Rostkowski (https://github.com/sr259)
+    - Hebron Bekele (https://github.com/hebronh)
+- **Start Date:** 25 Aug 2023
```
Comment (kate-holdener) [inline docs/droneworld/about.md:18]: @yashaswini-slu I think the start date is not right - this should be the start date of Dr. Agrawal's drone world project.

File: docs/droneworld/about.md (hunk 1 above)
[the remark this answers is not in the document]
Comment (kungfuchicken) [reply docs/droneworld/about.md]: i agree

**Comment by kungfuchicken** [comment]
There is a minor issue that the project directory should be named `project_droneworld` rather than `droneworld` so that the projects all sort next to each other in a directory.

**Review by kungfuchicken — APPROVED** [review]
lgtm;

File: docs/droneworld/about.md (hunk 1 above)
Comment (kungfuchicken) [reply docs/droneworld/about.md]: resolved

File: docs/droneworld/about.md (hunk 2 above)
Comment (kungfuchicken) [reply docs/droneworld/about.md]: resolved.

**Pull request merged**